docs/editor/debugging.md

```diff
@@ -319,6 +319,27 @@ Source maps can be generated with two kinds of inlining:
 
 VS Code supports both the **inlined source maps** and the **inlined source**.
 
+#### JavaScript Source Map Tips
+

```

Comment:
these were very early samples of basic express apps that were fully configured to get the best out of VS Code. I don't think we reference them anywhere.